member/src/main/resources/application.conf

```diff
@@ -0,0 +1,34 @@
+akka.actor.provider = cluster
+
+//FIXME - autocreate only for dev
+akka.actor.serialization-bindings{
+  "com.improving.app.member.utils.serialize.CborSerializable" = jackson-cbor
+}
```

Comment:
jackson-cbor (or preferably json) is a fine choice for persistent events in a database.  It is less viable than protobuf where backwards compatibility become important over cross-service boundaries.

Reply:
For now going with json-cbor